data/Anshul439.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Anshul Wadhwa",
+  "bio": "Pursuing Bachelor's in Computer Science",
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "http://www.linkedin.com/in/anshul-wadhwa",
+      "icon": "FaLinkedIn"
+    }
```

Please add the missing comma here.
And you're probably using an invalid icon here. It should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
    },
```